### PR TITLE
Fixed Interface for PDF Service and added new sql script

### PR DESCRIPTION
**Pull request by CodingVelocista**

### CODE REVIEW

File: embc-app/Controllers/ReferralsController.cs [hunk 1]
```diff
@@ -81,15 +80,15 @@ public async Task<IActionResult> Post(string registrationId, [FromBody] PostRequ
...
         public async Task<IActionResult> GetReferralPdfs([FromBody] ReferralsToPrint printReferrals)
-        {
-            var content = await referralsService.GetReferralHtmlPages(printReferrals);
+        {
+            var result = await referralsService.GetReferralPdfs(printReferrals);
 
-            if (string.IsNullOrWhiteSpace(content))
+            if (result == null)
             {
                 return NotFound(printReferrals.ReferralIds);
```
Comment (ytqsl) [inline embc-app/Controllers/ReferralsController.cs]: Need to format as Json

File: embc-app/Services/Referrals/IReferralsService.cs [hunk 2]
```diff
@@ -1,10 +1,13 @@
 using Gov.Jag.Embc.Public.ViewModels;
+using Microsoft.AspNetCore.Mvc;
```
Comment (ytqsl) [inline embc-app/Services/Referrals/IReferralsService.cs]: Make sure CodeMaid is cleaning up everytime you save a file

File: embc-app/Services/Referrals/IReferralsService.cs [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 using System.Threading.Tasks;
 
 namespace Gov.Jag.Embc.Public.Services.Referrals
 {
     public interface IReferralsService
     {
+        Task<byte[]> GetReferralPdfs(ReferralsToPrint printReferrals);
```
Comment (ytqsl) [inline embc-app/Services/Referrals/IReferralsService.cs]: Async?

File: embc-app/Services/Referrals/IReferralsService.cs [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
+
         Task<string> GetReferralHtmlPages(ReferralsToPrint printReferrals);
```
Comment (ytqsl) [inline embc-app/Services/Referrals/IReferralsService.cs]: Async?

File: embc-app/Services/Referrals/ReferralsService.cs [hunk 5]
```diff
@@ -8,16 +8,31 @@
 using Gov.Jag.Embc.Public.Utils;
 using System.Collections.Generic;
 using System.ComponentModel;
+using Microsoft.AspNetCore.Mvc;
```
Comment (ytqsl) [inline embc-app/Services/Referrals/ReferralsService.cs]: is it used?

File: embc-app/Controllers/ReferralsController.cs [hunk 6]
```diff
@@ -6,6 +6,7 @@
 using Microsoft.AspNetCore.Mvc;
 using System.Collections.Generic;
 using System.Linq;
+using System.Net;
```
Comment (ytqsl) [inline embc-app/Controllers/ReferralsController.cs]: Is this used in the code?

File: embc-app/Services/Referrals/IReferralsService.cs [hunk 7]
```diff
@@ -5,7 +5,9 @@ namespace Gov.Jag.Embc.Public.Services.Referrals
 {
     public interface IReferralsService
     {
-        Task<string> GetReferralHtmlPages(ReferralsToPrint printReferrals);
+        Task<byte[]> GetReferralPdfsAsync(ReferralsToPrint printReferrals);
+
+        Task<string> GetReferralHtmlPagesAsync(ReferralsToPrint printReferrals);
```
Comment (ytqsl) [inline embc-app/Services/Referrals/IReferralsService.cs]: is this part of the interface? Is it used?

File: embc-app/Controllers/ReferralsController.cs (hunk 6 above)
Comment (CodingVelocista) [reply embc-app/Controllers/ReferralsController.cs]: No that should have been removed in my sweep

File: embc-app/Services/Referrals/IReferralsService.cs (hunk 7 above)
Comment (CodingVelocista) [reply embc-app/Services/Referrals/IReferralsService.cs]: Should have removed that as well.

**Pull request merged**